.github/workflows/ci_build_scm_gnu9.4.yml

```diff
@@ -0,0 +1,113 @@
+name: CI test to build SCM
+
+on: [push, pull_request]
+
+jobs:
+  build-linux:
+
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+
+    # Environmental variables
+    env:
+      NFHOME:     /home/runner/netcdf-fortran
+      bacio_ROOT: /home/runner/bacio
+      sp_ROOT:    /home/runner/NCEPLIBS-sp
+      w3emc_ROOT: /home/runner/myw3emc
+
+    # Workflow steps
+    steps:
+
+    - name: Checkout SCM code (into /home/runner/work/ccpp-scm/)
+      uses: actions/checkout@v3
+
+    - name: Initialize submodules
+      run: git submodule update --init --recursive
+    
+    - name: Set up Python 3.8.5
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.8.5
+
+    - name: Add conda to system path
+      run: |
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        echo $CONDA/bin >> $GITHUB_PATH
+
+    - name: Install python libraries
+      run: |
+        conda install --yes -c conda-forge f90nml
+        conda install --yes -c conda-forge netCDF4
+
+    - name: Set compilers
+      run: |
+        echo "FC=gfortran-9" >> $GITHUB_ENV
+        echo "CC=gcc-9"      >> $GITHUB_ENV
+
+    - name: Install bacio library v2.4.1
```

Comment:
Do we really have to install these libraries every time? There's no way to do something similar as Python with `uses:`? Seems like a waste of cycles, but if that's what has to happen, then so be it. It'd be great if there was container support for CI.

Reply:
@grantfirl Good point. I will look into this further.

Reply:
@grantfirl Turns out you can cache installs in these workflows.[ I added a few more steps to save/cache some libraries.](https://github.com/dustinswales/ccpp-scm/blob/update_multirun/.github/workflows/ci_build_scm_gnu9.4.yml#L58)